spec/factories/transactions.rb

```diff
@@ -11,5 +11,9 @@
     trait :category_transactions do
       transactinable { |transaction| transaction.association(:category_transaction) }
     end
+
+    trait :between_categories_trans do
```

Comment:
Maybe we should not use `between_categories_trans` names. I think it's better to find more acceptable name instead of shortenings.

Reply:
@sinpai again I fixed it in the next PR =(